File: scala-spring/queryside-web/src/main/scala/net/chrisrichardson/eventstore/examples/bank/web/queryside/QuerySideWebConfiguration.scala
package net.chrisrichardson.eventstore.examples.bank.web.queryside

import net.chrisrichardson.eventstore.examples.bank.queryside.QuerySideConfiguration
import org.springframework.context.annotation._

@Configuration
@Import(Array(classOf[QuerySideConfiguration]))
@ComponentScan
class QuerySideWebConfiguration {

}

File: java-spring/accounts-query-side-web/src/main/java/net/chrisrichardson/eventstore/javaexamples/banking/web/queryside/QuerySideWebConfiguration.java
package net.chrisrichardson.eventstore.javaexamples.banking.web.queryside;

import net.chrisrichardson.eventstore.javaexamples.banking.backend.queryside.accounts.QuerySideAccountConfiguration;
import org.springframework.context.annotation.ComponentScan;
import org.springframework.context.annotation.Configuration;
import org.springframework.context.annotation.Import;

@Configuration
@Import({QuerySideAccountConfiguration.class})
@ComponentScan
public class QuerySideWebConfiguration  {


}
